### PR TITLE
Finish day 3 and go out with your friends to drink

### PR DESCRIPTION
**Pull request by Zephyrrus**

### CODE REVIEW

File: earth.planet [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-	"heaven": {"content": "The good things", "level": 9999},
- 	"earth": {"content": null, "level": 0, "features": ["day", "night", "sky", "water"]},
-	"hell": {"content": "scalies", "level": -9999},
+	"heaven": {"content": ["God", "Adam", "Eve", "snakes"], "level": 9999},
+ 	"earth": {"content": ["vegetation"], "level": 0, "features": ["day", "night", "sky", "water", "land"]},
+	"hell": {"content": ["scalies", "furries"], "level": -9999},
```
Comment (nepeat) [inline earth.planet]: furries in hell? nope

Comment (Zephyrrus) [reply earth.planet]: Where do you want to store them

**Comment by nepeat** [comment]
i demand justice for those furries stuck in hell

**Comment by Zephyrrus** [comment]
They will be moved to earth when I finalize it. 

You can't just put furries on a planet without animal can you?

**Pull request merged**